platform/ios/src/MGLMapView.mm

```diff
@@ -1975,7 +1975,7 @@ - (void)trackGestureEvent:(NSString *)gestureID forRecognizer:(UIGestureRecogniz
 
 - (void)showAttribution:(__unused id)sender
 {
-    NSString *title = NSLocalizedStringWithDefaultValue(@"SDK_NAME", nil, nil, @"Mapbox iOS SDK", @"Action sheet title");
+    NSString *title = NSLocalizedStringWithDefaultValue(@"SDK_NAME", nil, nil, @"Mapbox Maps SDK for iOS", @"Action sheet title");
```

Comment:
This change should be reflected in at least the base Localizable.strings file, if not all the translations as well.

Reply:
Oops, thanks! I don't know why I didn't commit those.